argoverse/data_loading/argoverse_tracking_loader.py

```diff
@@ -459,10 +480,14 @@ def get_image_at_timestamp(
         if log_id is None:
             log_id = self.current_log
         assert self.timestamp_image_dict is not None
+
+        if timestamp is None:
+            return None
+
```

Comment:
was there a specific reason we need this check? may a better one would be
```python
assert isinstance(timestamp, int), "query timestamp provided was not an integer"
```
since timestamp should never be none?

Reply:
I guess I'm using None timestamp to represent missing image, so it could happen in this case. (Line 206 for example) We could check it before calling the function, but I don't see much of a different either way?